<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>903</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1055</width>
    <height>903</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1055</width>
    <height>903</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Malgun Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>VTS 메타데이터 입력기</string>
  </property>
  <property name="locale">
   <locale language="Korean" country="SouthKorea"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>Ingest 타입</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1031</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="title">
     <string>소재 정보</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>영상구분</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>이벤트</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>76</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>인제스트 소스</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>제목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>24</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>폴더</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>카테고리</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>사용제한</string>
     </property>
    </widget>
    <widget class="QComboBox" name="subjectComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="eventComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="sourceComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="folderComboBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="categoryComboBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="restrictionComboBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="titleLineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>951</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>1031</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="title">
     <string>제작 정보</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>취재부서</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>영상기자</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>촬영장소</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>내용</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>취재기자</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
      </font>
     </property>
     <property name="text">
      <string>촬영일</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="videoDateWidget">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>291</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Malgun Gothic</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="contentTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>711</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="deptLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="journalistLineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="videoReporterLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="placeLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>파일 리스트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>560</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>630</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>690</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xmlCreateButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>820</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>XML 생성</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ingestTypeComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="fileListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>841</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>750</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
